framework/src/actions/SetupDebugAction.C

```diff
@@ -40,8 +40,12 @@ SetupDebugAction::validParams()
   params.addParam<bool>("show_mesh_meta_data", false, "Print out the available mesh meta data");
   params.addParam<bool>(
       "show_reporters", false, "Print out information about the declared and requested Reporters");
-  params.addParam<bool>(
+  params.addDeprecatedParam<bool>(
       "pid_aux",
+      "Add a AuxVariable named \"pid\" that shows the processors and partitioning",
+      "pid_aux is deprecated, use show_process_domains");
+  params.addParam<bool>(
+      "show_process_domains",
```

Comment:
This is an improvement. Does `show_processor_domains` make things even more clearer, given your description?

Reply:
well a process could be running on multiple processors (with threads) so maybe the description should change

Reply:
I would be in favor of adjusting the description if we leave the name as-is, that's fine

Reply:
done